libvast/src/system/node.cpp

```diff
@@ -467,7 +467,11 @@ node_state::spawn_command(const invocation& inv,
   VAST_TRACE_SCOPE("{}", inv);
   using std::begin;
   using std::end;
-  auto self = this_node;
+  auto* self = this_node;
+  if (self->state.tearing_down)
+    return caf::make_message(caf::make_error(caf::sec::request_receiver_down,
```

Comment:
Doesn't this error code have a special meaning inside CAF? We should be careful using these.

Also this is not an error per se, right? I think we should just return `caf::none` or `caf::no_error` here to indicate that nothing was spawned.